### PR TITLE
Add io.gitlab.LibreGames.jumpnbump

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
> platform multiplayer video game
- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** https://gitlab.com/LibreGames/jumpnbump/-/issues/35

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build com.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build com.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build com.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build io.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build io.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build io.gitlab.LibreGames.jumpnbump

File: appstream.patch [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+diff --git a/dist/jumpnbump.appdata.xml b/dist/jumpnbump.appdata.xml
+index c0a9627..4160549 100644
+--- a/dist/jumpnbump.appdata.xml
++++ b/dist/jumpnbump.appdata.xml
+@@ -27,7 +27,7 @@
+   </content_rating>
+   <screenshots>
+     <screenshot type="default" width="800" height="512">
+-      https://gitlab.com/LibreGames/jumpnbump/raw/master/dist/screenshot.png
++      <image>https://gitlab.com/LibreGames/jumpnbump/raw/master/dist/screenshot.png</image>
```
Comment (bbhtt) [inline appstream.patch:117]:
```suggestion
+      <image>https://gitlab.com/LibreGames/jumpnbump/-/raw/70854c4ca45d2ee951c4c8b2ad5afde94bdc9591/dist/screenshot.png</image>
```

File: appstream.patch [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+ 		$(DESTDIR)$(DATADIR)/applications/jumpnbump.desktop
+ 	install -m 644 dist/jumpnbump.png \
+-		$(DESTDIR)$(DATADIR)/icons/jumpnbump.png
++		$(DESTDIR)$(DATADIR)/icons/hicolor/128x128/apps/jumpnbump.png
+ 	install -m 644 dist/jumpnbump.6 $(DESTDIR)$(DATADIR)/man/man6/
+ 
+ #	$(MAKE) -C menu install
+diff --git a/dist/jumpnbump.png b/dist/jumpnbump.png
+index 16ec92b9bebf96e56023e3efe21b177f2a6c2b52..a90087c0fd47914923b4503ec05cb4acdab513c8 100644
+GIT binary patch
```
Comment (bbhtt) [inline appstream.patch:185]: What is this doing? Changing the icon size?

File: io.gitlab.LibreGames.jumpnbump.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: jumpnbump
+rename-desktop-file: jumpnbump.desktop
+rename-icon: jumpnbump
+rename-appdata-file: jumpnbump.appdata.xml
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
```
Comment (bbhtt) [inline io.gitlab.LibreGames.jumpnbump.yaml:12]: Does this work on native wayland?

File: appstream.patch (hunk 2 above)
Comment (Mailaender) [reply appstream.patch:185]: 96px → 128px

File: appstream.patch (hunk 1 above)
Comment (Mailaender) [reply appstream.patch:117]: I submitted https://gitlab.com/LibreGames/libregames.gitlab.io/-/merge_requests/1 instead.

File: io.gitlab.LibreGames.jumpnbump.yaml (hunk 3 above)
Comment (Mailaender) [reply io.gitlab.LibreGames.jumpnbump.yaml:12]: Yes

File: appstream.patch (hunk 1 above)
Comment (bbhtt) [reply appstream.patch:117]: Ok sure.

**Comment by bbhtt** [comment]
How do you move past the loading screen?

image

**Comment by bbhtt** [comment]
Ah ok that's where the game starts, didn't realise that. Seems fine.

**Comment by Mailaender** [comment]
> How do you move past the loading screen?

The game design is a bit confusing there. The AppStream metadata text explains it.

**Comment by Mailaender** [comment]
bot, build io.gitlab.LibreGames.jumpnbump

**Comment by Mailaender** [comment]
bot, build io.gitlab.LibreGames.jumpnbump

**Pull request closed without merging**